core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -106,19 +150,22 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     config.put(AdminClientConfig.CLIENT_ID_CONFIG, clientId)
     client = Admin.create(config)
 
-    val entity = new ClientQuotaEntity(Map(ClientQuotaEntity.CLIENT_ID -> clientId).asJava)
-    val configEntries = Map(QuotaConfigs.CONTROLLER_MUTATION_RATE_OVERRIDE_CONFIG -> 1.0, QuotaConfigs.CONSUMER_BYTE_RATE_OVERRIDE_CONFIG -> 3.0)
-    client.alterClientQuotas(Seq(new ClientQuotaAlteration(entity, configEntries.map {case (k, v) =>
-      new ClientQuotaAlteration.Op(k,v)}.asJavaCollection)).asJavaCollection).all.get
+    try {
+      val entity = new ClientQuotaEntity(Map(ClientQuotaEntity.CLIENT_ID -> clientId).asJava)
+      val configEntries = Map(QuotaConfigs.CONTROLLER_MUTATION_RATE_OVERRIDE_CONFIG -> 1.0, QuotaConfigs.CONSUMER_BYTE_RATE_OVERRIDE_CONFIG -> 3.0)
+      client.alterClientQuotas(Seq(new ClientQuotaAlteration(entity, configEntries.map { case (k, v) =>
+        new ClientQuotaAlteration.Op(k, v)
+      }.asJavaCollection)).asJavaCollection).all.get
 
-    TestUtils.waitUntilTrue(() => {
-      // wait for our ClientQuotaEntity to be set
-      client.describeClientQuotas(ClientQuotaFilter.all()).entities().get().size == 1
-    }, "Timed out waiting for quota config to be propagated to all servers")
+      TestUtils.waitUntilTrue(() => {
+        // wait for our ClientQuotaEntity to be set
+        client.describeClientQuotas(ClientQuotaFilter.all()).entities().get().size == 1
+      }, "Timed out waiting for quota config to be propagated to all servers")
 
-    val quotaEntities = client.describeClientQuotas(ClientQuotaFilter.all()).entities().get()
+      val quotaEntities = client.describeClientQuotas(ClientQuotaFilter.all()).entities().get()
 
-    assertEquals(configEntries,quotaEntities.get(entity).asScala)
+      assertEquals(configEntries, quotaEntities.get(entity).asScala)
+    } finally client.close(time.Duration.ZERO)
```

Comment:
this client is created normally (correct port), so it can be closed as usual. that means we don't need to add try-finally, as it will be closed by the `@AfterEach`

Reply:
ok, done.